src/components/ShoppingList/PopulatedList.js

```diff
@@ -1,15 +1,39 @@
-import React, { Fragment } from "react";
+import React, { Fragment, useState, useEffect } from "react";
 import { useCollection } from "react-firebase-hooks/firestore";
 import { db } from "../../lib/firebase";
 
 const PopulatedList = () => {
-  const [value, loading, error] = useCollection(
+  const [listData] = useCollection(
     db.collection(localStorage.getItem("token")),
     {
       snapshotListenOptions: { includeMetadataChanges: true },
     },
   );
 
+  const [filterValue, setFilterValue] = useState("");
+  const [filteredList, setFilteredList] = useState([]);
+
+  useEffect(() => {
+    if (listData) {
+      let filtered = listData.docs.filter((doc) => {
+        return doc
+          .data()
+          .itemName.toLowerCase()
+          .includes(filterValue.toLowerCase());
```

Comment:
Love that you're adding the extra check for "if" there is a list and then comparing each list item to the search term by lowercasing both to make sure they're the same!